packages/components/loading/src/loading-provider.tsx

```diff
@@ -358,11 +358,11 @@ const getOverlayStyle = (
   right: 0,
   bottom: 0,
   left: 0,
-  zIndex: "beerus",
+  zIndex: 9999,
   bg: type === "fill" ? ["white", "black"] : "blackAlpha.600",
```

Comment:
Please refer to `colors.ts` for reference.
```suggestion
  bg: type === "fill" ? ["#fbfbfb", "#141414"] : "rgba(0, 0, 0, 0.48)",
```